docs/ManualDeploymentGuideScalarDLOnAzure-1.md

```diff
@@ -0,0 +1,217 @@
+* Configure kubectl to connect to your Kubernetes cluster using the `az aks get-credentials` command. 
+
+    ```console
+    $ az aks get-credentials --resource-group myResourceGroup --name myAKSCluster
+    ```
+
+## Step 4. Install Scalar DL
+
+After creating a Kubernetes cluster next step is to deploy Scalar DL into the AKS cluster. 
+This section shows how to install Scalar DL to the AKS cluster with [Helm charts](https://github.com/scalar-labs/helm-charts).
+
+### Prerequisites
+
+Install Helm on your bastion to deploy helm-charts:
+
+* [Helm](https://helm.sh/docs/intro/install/): helm command-line tool to manage releases in the AKS cluster. In this tutorial, it is used to deploy Scalar DL and Schema loading helm charts to the AKS cluster. 
+   Helm version 3.2.1 or latest is required. 
+
+### Requirements
+
+* You must have the authority to pull `scalar-ledger` and `scalardl-schema-loader` container images.
+* You must confirm that the replica count of the ledger and envoy pods in the `scalardl-custom-values.yaml` file is equal to the number of nodes in the `scalardlpool`.
+
+### Steps
+
+1. Download the following Scalar DL configuration files from [the scalar-kubernetes repository](https://github.com/scalar-labs/scalar-kubernetes/tree/master/conf). Note that they are going to be versioned in the future, so you might want to change the branch to use a proper version.
+    * scalardl-custom-values.yaml
+    * schema-loading-custom-values.yaml
+
+2. Update the database configuration in `scalarLedgerConfiguration` and `schemaLoading` sections as specified in [Set up a database guide](./SetupAzureDatabase-1.md#Configure-Scalar-DL).
+
+3. Create the docker-registry secret for pulling the Scalar DL images from the GitHub Packages.
+    
+   ```console
+    $ kubectl create secret docker-registry reg-docker-secrets --docker-server=ghcr.io --docker-username=<github-username> --docker-password=<github-personal-access-token>
+    ```
+   
+4. Run the Helm commands on the bastion to install Scalar DL on AKS.
+    
+   ```console
+    # Add Helm charts 
+    $ helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
+    
+    # List the Scalar charts.
+    $ helm search repo scalar-labs
+    
+    # Load Schema for Scalar DL install with a release name `load-schema`
+    $ helm upgrade --install load-schema scalar-labs/schema-loading --namespace default -f schema-loading-custom-values.yaml
+    
+    # Install Scalar DL with a release name `my-release-scalardl`
+    $ helm upgrade --install my-release-scalardl scalar-labs/scalardl --namespace default -f scalardl-custom-values.yaml
+    ```
+
+Note:
+
+* The same commands can be used to upgrade the pods.
+* Release name `my-release-scalardl` can be changed as per your convenience.
+* `helm ls -a` command can be used to list currently installed releases.
+* You should confirm the load-schema deployment has been completed with `kubectl get po -o wide` before installing Scalar DL.
```

Comment:
Since the shorthands have been removed elsewhere,

```suggestion
* You should confirm the load-schema deployment has been completed with `kubectl get pods -o wide` before installing Scalar DL.
```

Reply:
Thank you for the suggestion, fixed in c496abd